examples/cms-prepr/apollo-client.js

```diff
@@ -0,0 +1,8 @@
+import { ApolloClient, InMemoryCache } from '@apollo/client'
+
```

Comment:
Any reason in particular to use apollo here?

Reply:
Let's use `fetch` instead 👍

Reply:
I believe in the old version we used our own SDK, but this was deprecated so we opted for a different package that a lot of our customers use. I have now changed it to use fetch instead.